persistence-modules/spring-jpa-3/src/test/java/com/baeldung/jpa/subtypes/JPAInheritanceSingleTableTest.java

```diff
@@ -0,0 +1,87 @@
+package com.baeldung.jpa.subtypes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.baeldung.jpa.subtypes.entity.ContractEmployee;
+import com.baeldung.jpa.subtypes.entity.Employee;
+import com.baeldung.jpa.subtypes.entity.PermanentEmployee;
+import com.baeldung.jpa.subtypes.repository.ContractEmployeeRepository;
+import com.baeldung.jpa.subtypes.repository.EmployeeRepository;
+import com.baeldung.jpa.subtypes.repository.PermanentEmployeeRepository;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = SubTypeJPAApplication.class)
+class JPAInheritanceSingleTableTest {
```

Comment:
We follow https://google.github.io/styleguide/javaguide.html for naming conventions
```suggestion
class JpaInheritanceSingleTableTest {
```